thrust/thrust/type_traits/remove_cvref.h

```diff
@@ -1,43 +0,0 @@
-/*
```

Comment:
Is it intended that `thrust::remove_cref` is piggy-backed? Might be worth a mention in the PR for posterity.

Reply:
I was debating that and decided to be lazy :(